docs/src/_includes/components/language-switcher.html

```diff
@@ -14,9 +14,12 @@
         <select name="language selector" id="language-select" aria-describedby="language-infobox"
             class="c-custom-select switcher__select">
             {% for key, other_site in sites %}
-            <option value="{{ other_site.language.code }}" data-url="https://{{ other_site.hostname }}" {% if
-                site.language.code==other_site.language.code %} selected {% endif %}>
-                {{ other_site.language.flag }} {{ other_site.language.name }}
+            <option
+                value="{{ other_site.language.code }}"
```

Comment:
This should ideally not work in localhost? Is this expected?

Reply:
Do you mean to hide it locally?

Reply:
Just asking if this would work in local as well?